data/net/minecraft/world/level/chunk/ChunkGenerator.mapping

```diff
@@ -1,3 +1,67 @@
 CLASS net/minecraft/world/level/chunk/ChunkGenerator
+	METHOD <init> (Lnet/minecraft/world/level/biome/BiomeSource;Lnet/minecraft/world/level/biome/BiomeSource;Lnet/minecraft/world/level/levelgen/StructureSettings;J)V
+		ARG 1 source
```

Comment:
Still not sold on `source vs, `biomeSource`

See: lower down in this class, there's a distinction made between `structureManager`, `templateManager`, but then occasional references to `manager` as well... which is neither of those, it's a `biomeManager`.

I think both should be prefixed with `biome` for consistency + readability.